hometask1/foo.rb

```diff
@@ -0,0 +1,46 @@
+class Foo
+  @@counter = 0
+  @instance_counter = 0
+
+  def initialize
+    self.class.counter += 1
+    @instance_counter = Foo.counter
+  end
+
+  def self.counter=value
+    @@counter=value
+  end
+
+  def self.counter
+    @@counter
+  end
+
+  def my_instance
+  if @instance_counter%2==0
+      p ':eval'
```

Comment:
Please, don't print result. Just **return** it.
You can print result of the method in last lines, where examples.